src/pages/administration/datasources/webuntis/new.vue

```diff
@@ -51,6 +51,7 @@
 						label="Passwort"
 						:placeholder="'WebUntis Passwort'"
 						class="mt--md"
+						autocomplete="new-password"
```

Comment:
I think 
```suggestion
						autocomplete="off"
```
is semanticly better suited here, because new-password seems to indicate that submitting this form updates the currently saved login credentials.